db-setup/local-mariadb-docker-setup.sh

```diff
@@ -18,8 +18,8 @@ n=0
 until [ $n -ge 15 ]
 do
   sleep 5
-  ( docker exec -i dittodb-mariadb mysql -pr2N5y7V* ) < db-setup/mariadb-reset.sql && break
+  ( docker exec -i dittodb-mariadb mariadb -pr2N5y7V* ) < db-setup/mariadb-reset.sql && break
```

Comment:
I assume you got an error when launchign the docker container that complained that the command `mysql` was not found, correct?